app/stylesheets/main.scss

```diff
@@ -195,7 +195,7 @@ html {
 
 ol *,
 li * {
-    font-family: inherit;
+    font-family: inherit !important;
```

Comment:
This re-adds `!important`? Did we want this removed as per the previous PR?

Reply:
Hm I didn't see that this file had a conflict when merging, but will fix.